src/utils/passportStrategyCallback.js

```diff
@@ -0,0 +1,20 @@
+import database from '../models';
```

Comment:
use `models` instead of `database` for consistency

Reply:
Okay. On it.